app/controllers/browse_controller.rb

```diff
@@ -4,11 +4,7 @@ class BrowseController < ApplicationController
 
   def index
     @page = MainstreamBrowsePage.find("/browse")
-    set_slimmer_artefact_headers({
-      title: "browse",
-      section_name: @page.title,
-      section_link: @page.base_path,
-    })
+    set_slimmer_artefact_headers
```

Comment:
Yup, it calls `set_slimmer_headers(format: 'browse')` regardless of presence/absence of args, but it sends an empty hash to `set_slimmer_dummy_artefact` (which sets the data to be used for breadcrumbs in Slimmer),.